src/index.html

```diff
@@ -1,13 +1,61 @@
-<!doctype html>
+ <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+        content="width=device-width, initial-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="block">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__container">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="card"
+          >
+          <h2 class="card__title">
+             APPLE A1419 iMac 27" Retina<br>
+             5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="card__code">Product code: 195434</p>
+          <div class="card__stars-reviews">
+            <div class="stars">
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+              <div class="stars__stars"></div>
+            </div>
+            <p class="card__reviews">
+              Reviews: 5
+            </p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">
+               Price:
+            </p>
+            <p class="card__price-price">
+               $2,199
+            </p>
+          </div>
+          <div class="card__button">
+            <a
+              class="card__button-linc"
+              data-qa="hover"
+              href="#"
+              title="BUY">
+                BUY
```

Comment:
```suggestion
              title="BUY"
            >
              Buy
```
use `text-transform`